libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -178,16 +181,18 @@ class RSCGovFloatParam {
     static const struct AP_Param::GroupInfo var_info[];
 
     void set_gov_enable(int8_t setenable) {enable = setenable; }
-    int16_t get_setpoint() { return setpoint; }
+    int16_t get_reference() { return reference; }
+    float get_range() { return range; }
     float get_disengage() { return disengage; }
-    float get_droop_setting() { return droop_setting; }
+    float get_droop_response() { return droop_response; }
     float get_tc() { return tc; }
 
 private:
     AP_Int8   enable;
-    AP_Float  disengage;    // sets the throttle percent where the governor disengages for return to flight idle
-    AP_Int16  setpoint;     // sets rotor speed for governor
-    AP_Float  droop_setting;// governor response to droop under load
-    AP_Float  tc;           // governor throttle curve weighting, range 50-100%
+    AP_Int16  reference;      // sets rotor speed for governor
+    AP_Float  range;          // RPM range +/- governor rpm reference setting where governor is operational
```

Comment:
maybe "range" should be "rpm_range" or "range_rpm"?

Reply:
This would be an easy enough change. But do we really want to make the variable names longer when the variable is declared as '_governor_range' with a clear explanation in the code comment?